### PR TITLE
Set datepicker day

### PR DESCRIPTION
**Pull request by taltas**
| Q               | A
| --------------- | ---
| Bug fix?        | no|yes
| New feature?    | no|yes
| BC breaks?      | no|yes
| Related tickets | fixes #X, partially #Y, mentioned in #Z
| License         | MIT

### CODE REVIEW

File: js/bootstrap-datepicker.js [hunk 1]
```diff
@@ -201,6 +201,9 @@
 			}
 			o.multidateSeparator = String(o.multidateSeparator);
 
+			// We need to set the default here as the default option values are set before the DOM has loaded,
+			// meaning currentUser hasn't loaded before it's attempted to be used
+			o.weekStart = window.currentUser.startDay || 0,
```
Comment (mbillard) [inline js/bootstrap-datepicker.js]: Is 0 Monday? Is that what it defaults to at the moment?

Comment (mbillard) [reply js/bootstrap-datepicker.js]: Also, it feels wrong to have code that references currentUser in this library? Can't we always set `weekStart` when we call this datepicker instead?

Comment (taltas) [reply js/bootstrap-datepicker.js]: From the amount of files I think it is going to take some effort to decouple this.

Comment (mbillard) [reply js/bootstrap-datepicker.js]: Ok, can we at least check if currentUser is present, just in case. `window.currentUser?.startDay`

**Pull request merged**